### PR TITLE
SDN-5346: image, mirror kubevirt fedora guest image

### PR DESCRIPTION
**Pull request by qinqon**
The ovn-kubernetes test to check persistent IPs during live migration use kubevirt to start VMs, this change add the guest image for those VMs.

Needed for:
- https://github.com/openshift/origin/pull/29199

### CODE REVIEW

**Comment by qinqon** [comment]
@bertinatto @tkashem can you take a look ?

File: test/extended/util/image/image.go [hunk 1]
```diff
@@ -55,6 +55,9 @@ var (
 		// tests will fail (vendor/k8s.io/kubernetes/test/utils/image/manifest.go)
 		"registry.k8s.io/e2e-test-images/agnhost:2.52": 1,
 		"registry.k8s.io/e2e-test-images/nginx:1.15-4": 21,
+
+		// used by kubeviert test to start fedora VMs
```
Comment (maiqueb) [inline test/extended/util/image/image.go:59]:
```suggestion
		// used by KubeVirt test to start fedora VMs
```

**Review by maiqueb — APPROVED** [review]
just caught a typo

**Review by maiqueb — APPROVED** [review]
/lgtm

**Comment by qinqon** [comment]
/cc @bertinatto @ingvagabund @soltysh @tkashem

Can you take a view to see if it's ok for origin ? We needed this to GA a OCP feature related to ovn-kubernetes primary UDN.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Comment by maiqueb** [comment]
@qinqon set the [SDN-5346](https://issues.redhat.com/browse/SDN-5346) user story instead of no-jira.

**Pull request merged**